CONTRIBUTING.md

```diff
@@ -25,7 +25,7 @@ To build Earthly from source for your target system, use
     ./earthly +for-darwin-m1
     ```
 
-This builds the earthly binary in `./build/<platform>/amd64/earthly` and also the buildkitd image.
+This builds the earthly binary in `./build/$(uname|tr A-Z a-z)/$(uname -m)/earthly` and also the buildkitd image.
```

Comment:
This isn't entirely accurate, as `uname -m` can vary a lot: `X86_64` vs `amd64`, or `arm64` vs `aarch64`.

Maybe just list out the three common options?

```
* ./build/linux/amd64/earthly
* ./build/darwin/amd64/earthly
* ./build/darwin/arm64/earthly
```

Reply:
I can't think of a remotely good way to write this. I'd almost argue in favor of:

```suggestion
This builds the earthly binary in `./build/*/*/earthly` and also the buildkitd image.
```

I'm assuming that on average you only have a single flavor.

Reply:
Hah! This works surprisingly well :-)

I think it makes sense! I would still list those three popular possibilities the first time we say `./build/*/*/earthly`, but then just use `./build/*/*/earthly` in the rest of the doc.